examples/website/src/shared/route/login.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useState, useContext, FormEvent, useEffect} from 'react'
+import {RouteActionType} from '@karma.run/react'
+
+import {useRouteDispatch, matchRoute, useRoute, PageRoute} from './routeContext'
+import {AuthDispatchContext, AuthDispatchActionType} from '../authContext'
+
+import {LocalStorageKey} from '../utility'
+import {gql, useMutation} from '@apollo/client'
+
+const AuthQuery = gql`
+  mutation CreateSession($email: String!, $password: String!) {
+    createSession(email: $email, password: $password) {
+      user {
+        email
+      }
+      token
+    }
+  }
+`
+
+export function Login() {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+  const {current} = useRoute()
+
+  const authDispatch = useContext(AuthDispatchContext)
+  const routeDispatch = useRouteDispatch()
+
+  const [authenticate, {loading, error}] = useMutation(AuthQuery)
+
+  useEffect(() => {
+    if (error) {
+      alert(error)
```

Comment:
Maybe also add this error to the pino-logger?

Reply:
there is no pino-logger in the website